samples/client/petstore/java/apache-httpclient/src/test/java/org/openapitools/client/model/SpecialCatAllOfTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * OpenAPI Petstore
+ * This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package org.openapitools.client.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+
+/**
+ * Model tests for SpecialCatAllOf
+ */
+public class SpecialCatAllOfTest {
+    private final SpecialCatAllOf model = new SpecialCatAllOf();
+
+    /**
+     * Model tests for SpecialCatAllOf
+     */
+    @Test
+    public void testSpecialCatAllOf() {
+        // TODO: test SpecialCatAllOf
```

Comment:
Adding before merging?

Reply:
I didn't plan to. It was created by the generator. It looks like most of the generated tests don't have any implementations in them. Is there anything that you think I should do with it?